diff --git a/dist/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js b/dist/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js
index 91a62a8..d13d74c 100644
--- a/dist/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js
+++ b/dist/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js
@@ -201,21 +201,27 @@ class ReactFlightWebpackPlugin {
                                         (filePathToModuleMetadata[module] = {
                                             id,
                                             chunks,
+                                            css: cssChunks,
                                             name: "*"
                                         }));
                         }
                         const chunks = [];
+                        const cssChunks = [];
                         chunkGroup.chunks.forEach(function (c) {
+                            let jsRecorded = false;
                             var _iterator = _createForOfIteratorHelper(c.files), _step;
                             try {
                                 for (_iterator.s(); !(_step = _iterator.n()).done;) {
                                     const file = _step.value;
-                                    if (!file.endsWith(".js"))
-                                        break;
                                     if (file.endsWith(".hot-update.js"))
-                                        break;
-                                    chunks.push(c.id, file);
-                                    break;
+                                        continue;
+                                    if (!jsRecorded && file.endsWith(".js")) {
+                                        chunks.push(c.id, file);
+                                        jsRecorded = true;
+                                    }
+                                    else if (file.endsWith(".css")) {
+                                        cssChunks.push(file);
+                                    }
                                 }
                             }
                             catch (err) {
