diff --git a/build/common.gypi b/build/common.gypi
index 3a59639..365178a 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -376,7 +376,8 @@
           }],
           ['OS=="linux" or OS=="freebsd" or OS=="openbsd" or OS=="netbsd"', {
             'cflags': [ '-Wall', '<(werror)', '-W', '-Wno-unused-parameter',
-                        '-Wnon-virtual-dtor', '-Woverloaded-virtual' ],
+                        '-Wnon-virtual-dtor', '-Woverloaded-virtual',
+                        '-Wno-unused-local-typedefs' ],
           }],
           ['OS=="linux" and v8_enable_backtrace==1', {
             # Support for backtrace_symbols.
diff --git a/build/standalone.gypi b/build/standalone.gypi
index 125c5bf..32eaf85 100644
--- a/build/standalone.gypi
+++ b/build/standalone.gypi
@@ -98,7 +98,8 @@
       'target_defaults': {
         'cflags': [ '-Wall', '<(werror)', '-W', '-Wno-unused-parameter',
                     '-Wnon-virtual-dtor', '-pthread', '-fno-rtti',
-                    '-fno-exceptions', '-pedantic' ],
+                    '-fno-exceptions', '-pedantic',
+                    '-Wno-unused-local-typedefs' ],
         'ldflags': [ '-pthread', ],
         'conditions': [
           [ 'OS=="linux"', {
