diff --git a/build/common.gypi b/build/common.gypi
index 3a59639..594abe4 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -173,7 +173,6 @@
           [ 'v8_use_arm_eabi_hardfloat=="true"', {
             'defines': [
               'USE_EABI_HARDFLOAT=1',
-              'CAN_USE_VFP2_INSTRUCTIONS',
             ],
             'target_conditions': [
               ['_toolset=="target"', {

